server/test/unit/app.test.js

```diff
@@ -70,4 +73,131 @@ describe('POST /API/nlp', function () {
             .send({dataset: "human_resources__core_dataset", input: "Plot a histogram of age", history: []})
             .expect(200, done);
     });
-});
\ No newline at end of file
+});
+
+describe('Test Simple Plot Functions', function () {
+    it('should render ok',async function () {
```

Comment:
could you please give the single tests better names (better than every time `should render ok`)

Reply:
oh ofc, i thought that was just a convention